service/models.py

```diff
@@ -31,27 +32,31 @@ class Inventory(db.Model):
     app = None
 
     # Table Schema
-    id = db.Column(db.Integer, primary_key=True)
+    pid = db.Column(db.Integer, primary_key=True)
+    condition = db.Column(db.String(63), primary_key=True)
     name = db.Column(db.String(63), nullable=False)
     quantity = db.Column(db.Integer, default=1)
-    description = db.Column(db.String(63), default='')
+    restock_level = db.Column(db.Integer)
+    available = db.Column(db.Integer)
 
-    def __init__(self,id:int, name: str, quantity: int, description: str=''):
-        """Constructor for Item in Inventory""" 
-        self.id = id
+    def __init__(self, pid: int, condition: str, name: str, quantity: int, restock_level: int, available: int):
+        """Constructor for Item in Inventory"""
+        self.pid = pid
+        self.condition = condition
         self.name = name
         self.quantity = quantity
-        self.description = description
+        self.restock_level = restock_level
+        self.available = available
 
     def __repr__(self):
-        return "<Inventory %r id=[%s], Item name=[%s], Item Quantity=[%s], Item Description=[%s]>" % (self.id, self.name, self.quantity, self.description)
+        return "<Inventory Product id=[%s], Condition=[%s], Name=[%s], Quantity=[%s], Restock_level=[%d], Available=[%d]>" % (self.pid, self.condition, self.name, self.quantity, self.restock_level, self.available)
 
     def create(self):
         """
         Creates a Inventory to the database
         """
         logger.info("Creating %s", self.name)
-        self.id = None  # id must be none to generate next primary key
+        self.pid = None  # pid must be none to generate next primary key
```

Comment:
remove this, as it doesn't make sense to have pid as self-sequencing

Reply:
Correct. Thanks for pointing that out.